.github/workflows/deploy.yaml

```diff
@@ -151,11 +151,10 @@ jobs:
         run: |
           docker-compose build
           sudo npm install -g git+https://github.com/AvadoDServer/AVADOSDK.git
```

Comment:
can we use a versioned url? like git+https://github.com/AvadoDServer/AVADOSDK.git#commit